util/delete_entities.py

```diff
@@ -19,8 +18,8 @@
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--workspace-namespace', required=True)
-    parser.add_argument('--workspace-name', required=True)
+    parser.add_argument('workspace_namespace')
```

Comment:
Why remove `required=True` ?

Reply:
`required` is only a permitted param for flags, I switched these to arguments which seems more logical to me (and it's how the documentation says to run it). Before removing the `--` and `required=True`, this actually had to be run as:
```
python util/delete_entities.py  --workspace-namespace=NAMESPACE --workspace-namespace=NAME
```

The arguments are still "required", in the sense that if you try to just run `python util/delete_entities.py` you get:
```
usage: delete_entities.py [-h] workspace_namespace workspace_name
delete_entities.py: error: too few arguments
```

Reply:
Makes sense, thanks